addons/l10n_zm_account/data/template/account.tax.group-zm.csv

```diff
@@ -0,0 +1,4 @@
+id,name,country_id,"tax_receivable_account_id","tax_payable_account_id"
+"zm_tax_group_0","VAT 0%","base.zm","zm_account_8310000","zm_account_9520000"
+"zm_tax_group_16","VAT 16%","base.zm","zm_account_8310000","zm_account_9520000"
+"zm_tax_group_16_wh","Withholding VAT","base.zm","zm_account_8310000","zm_account_9520000"
```

Comment:
would be weird that these are the same as L2-3

Reply:
@jco-odoo I didn't get what is weird here? That the Tax payable and receivable account for the withholding VAT Tax Group is the same ad the ones of other Tax Group? At the end the tax payable/receivable accounts represent respectively the tax due to/owed by the authorities at the closing entry. I haven't seen in Zambia COA dedicated accounts for closing withholding VAT. Do you have examples in other localisations where we make the distinction?

Reply:
@Laura-labe Weirdly enough, I can not find an example in the other localizations immediately, but I think that all the other localizations with withholding are wrong, but for that, we ideally need the different tax closings.